Userland/Libraries/LibWeb/Page/Frame.h

```diff
@@ -101,6 +103,12 @@ class Frame : public TreeNode<Frame> {
 
     void did_edit(Badge<EditEventHandler>);
 
+    void register_frame_nesting(URL url);
+    bool is_frame_nesting_allowed(URL url);
+
+    void set_frame_nesting_levels(const HashMap<URL, size_t> frame_nesting_levels) { m_frame_nesting_levels = frame_nesting_levels; };
```

Comment:
Please use `move()` to avoid the `HashMap` copy here:
```cpp
void set_frame_nesting_levels(HashMap<URL, size_t> frame_nesting_levels) { m_frame_nesting_levels = move(frame_nesting_levels); }
```